src/components/Pagination/index.scss

```diff
@@ -1,3 +1,7 @@
 .pagination-container {
   margin-bottom: 30px;
 }
+
+.is-invisible {
```

Comment:
we can rename to is-hidden. is invisible suggests more the css property visibility: none